compliance_checker/protocols/cdl.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+'''
+compliance_checker/protocols/cdl.py
+'''
+import os
+
+
+def is_cdl(filename):
+    '''
+    Quick check for .cdl ascii file
+
+    Example:
+        netcdf sample_file {
+        dimensions:
+            name_strlen = 7 ;
+            time = 96 ;
+        variables:
+            float lat ;
+                lat:units = "degrees_north" ;
+                lat:standard_name = "latitude" ;
+                lat:long_name = "station latitude" ;
+        etc...
+
+    :param str filename: Absolute path of file to check
+    :param str data: First chuck of data from file to check
+    '''
+    if os.path.splitext(filename)[-1] != '.cdl':
```

Comment:
This will throw away valid `cdl` files that are not with the proper extension. I am OK with that behavior (actually I like it) but it should be documented to avoid confusion.